topics.md

```diff
@@ -0,0 +1,63 @@
+# Collection of topics
+
+Here we collect all the topics for future meetups with potential speekers/lecturer.
+
+### Into OTP
+What is OTP and how does it relate to elixir and erlang?
+
+Speeker:
+
+
+### TDD
+How to implement test driven design principles in elixir?
+
+Speeker:
+
+### DDD
+How to implement domain driven design principles in elixir?
+How does DDD relate to (Phoenix) contexts?
+
+Speeker:
+
+### Elixir's type system and dialyzer
+How does the type system in the elixir world work?
+
+Speeker:
+
+### Scaling and updating
+How to run and release an elixir application at scale
+
+Speeker:
+
+### How to structure elixir apps
+What options are there to structure elixir applications
+
+Speeker:
+
+### Macros and the compiler
+What is elixir at its core?
+
+Speeker: Raphael
+
+### Introduction to the Ash framework
```

Comment:
"ash" (consistently, either capitalized or not)

Reply:
From what i saw its called "Ash" not "ash"